content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,183 @@
+and process them for storage and retrieval. Example providers include Zipkin, Jaeger, Lightstep,
+Datadog, and Apache SkyWalking.
+
+For Istio, tracing providers are configured for use within the mesh via `MeshConfig`. To configure new
+providers to use in tracing, edit the `MeshConfig` for your mesh via:
+
+{{< text bash >}}
+$ kubectl -n istio-system edit configmap istio
+{{</ text >}}
+
+The full set of configuration options is described in the [reference docs for MeshConfig](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig).
+Typical configuration includes service address and port for the provider, as well as establishing
+a limit on max tag length supported by the provider.
+
+Each configured provider *must* be named. That name will be used to refer to the provider in the
+Telemetry API.
+
+An example set of provider configuration in `MeshConfig` is:
+
+{{< text yaml >}}
+data:
+    mesh: |-
+       # The following content defines two example tracing providers.
+        extensionProviders:
+        - name: "zippy-kin"
+          zipkin:
+            service: "zipkin.istio-system.svc.cluster.local"
+            port: 9411
+            maxTagLength: 56
+        - name: "stacky-driver"
+          stackdriver:
+            maxTagLength: 256
+        # A mesh default for provider can be specified to simplify Telemetry resources
+        defaultProviders:
+            tracing: "stacky-driver"
+{{</ text >}}
+
+### Configuring mesh-wide tracing behavior
+
+Telemetry API resources inherit from the root configuration namespace for a mesh, typically `istio-system`. To configure
+mesh-wide behavior, add a new (or edit the existing) `Telemetry` resource in the root configuration namespace.
+
+Here is an example configuration that uses the provider configuration from the prior section:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers: # only a single tracing provider can be selected at this time
+    - name: zippy-kin
+    customTags:
+      foo:
+        literal:
+          value: bar
+    randomSamplingPercentage: 100
+{{</ text >}}
+
+This configuration overrides the default provider from `MeshConfig`, setting the mesh default to be the `"zippy-kin"`
+provider. It also sets the mesh-wide sampling percentage to be `100`, and configures a tag to be added to all trace
+spans with a name of `foo` and a value of `bar`.
+
+### Configuring namespace-scoped tracing behavior
+
+To tailor the tracing behavior for individual namespaces, add a `Telemetry` resource to the desired namespace. Any tracing
+fields specified in the namespace resource will completely override the inherited field configuration from the configuration hierarchy.
+
+For example:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: namespace-override
```

Comment:
Do we need to have a specific name here or are multiple selector less resources merged?

Reply:
We don't need a specific name, but multiple selector-less resources are not valid configuration.